README.md

```diff
@@ -176,25 +176,30 @@ search for a Python interpreter matching that version in the following order:
 
 ### Installing into arbitrary Python environments
 
-Since uv has no dependency on Python, it can even install into virtual environments other than
+Since uv has no dependency on Python, it can install into virtual environments other than
 its own. For example, setting `VIRTUAL_ENV=/path/to/venv` will cause uv to install into
-`/path/to/venv`, no matter where uv is installed.
-
-uv can also install into arbitrary, even non-virtual environments by providing a `--python` argument
-to `uv pip sync` or `uv pip install`. For example, `uv pip install --python=/path/to/python` will
-install into the environment linked to the `/path/to/python` interpreter.
-
-For convenience, `uv pip install --system` will install into the system Python environment, as an
-approximate shorthand for, e.g., `uv pip install --python=$(which python3)`. Though we generally
```

Comment:
Moved `--python=$(which python3)` to previous paragraph, since no longer equivalent to `--system`.

Reply:
It's only saying it's an "approximate" shorthand. I think it's rare that your `python` is a shim which automatically activates environments on invocation? Just responding to your comment though I might be okay with where it is now.

Reply:
> It's only saying it's an "approximate" shorthand.

Sure, I get that. I guess what was confusing for me was than in uv<0.2, `--system` seemed more or less equivalent to `--python=$(which python3)`, whereas in uv>=0.2, this was no longer the case, at least for my use-case. Moreover, it's not currently clear in the text on `main` under which conditions the equivalence holds.

> I think it's rare that your `python` is a shim which automatically activates environments on invocation?

Hmm, as a pyenv-virtualenv user, this is always the case for me, but I'm not sure how prevalent this is, writ large.

I guess the main suggestion here is to provide clarity as to what `--system` does: perhaps including how it may be similar to `--python=$(which python3)` and how it differs.